a3p-integration/proposals/b:wallet-factory2/package.json

```diff
@@ -0,0 +1,21 @@
+{
+    "agoricProposal": {
+        "type": "/agoric.swingset.CoreEvalProposal",
+        "source": "build",
+        "buildScript": "../packages/vats/scripts/build-wallet-factory2-upgrade.js"
+    },
+    "type": "module",
+    "license": "Apache-2.0",
+    "dependencies": {
+        "@agoric/synthetic-chain": "^0.0.3",
```

Comment:
if you use `0.0.4-2` then you don't need `core-eval-support.js` and most of `performActions.js`. 

you'll still need `eval.sh` because you're making an extra submission distinct from the `agoricProposal` one.

Reply:
`3836e70a` takes us back to 0.0.3, so I'm going to assume for now that the suggestions about reducing `performActions.js` won't apply until a later branch.